client/src/components/ContentNavigator/ContentDataProvider.ts

```diff
@@ -317,6 +356,181 @@ class ContentDataProvider
   public createDirectory(): void | Thenable<void> {
     throw new Error("Method not implemented.");
   }
+
+  public reveal(item: ContentItem): void {
+    this._treeView.reveal(item, {
+      expand: true,
+      select: false,
+      focus: false,
+    });
+  }
+
+  private async handleContentItemDrop(
+    target: ContentItem,
+    item: ContentItem
+  ): Promise<void> {
+    let success = false;
+    let message = Messages.FileDropError;
+    if (item.flags.isInRecycleBin) {
+      message = Messages.FileDragFromTrashError;
+    } else if (isReference(item)) {
+      message = Messages.FileDragFromFavorites;
+    } else if (target.type === TRASH_FOLDER_TYPE) {
+      success = await this.recycleResource(item);
+    } else if (target.type === FAVORITES_FOLDER_TYPE) {
+      success = await this.addToMyFavorites(item);
+    } else {
+      // First we try to use target.uri to move our files. This works in instances where the
+      // folder is not a root folder. If this fails, then our best guess is that the target
+      // folder is a root folder, so we uri the "self" link instead
+      if (target.uri) {
+        success = await this.model.moveTo(item, target.uri);
+      }
+
+      const targetUri = getLink(target.links, "GET", "self")?.uri;
```

Comment:
it's same to utils.getResourceIdFromItem()

Reply:
Addressed in https://github.com/sassoftware/vscode-sas-extension/pull/268/commits/540d92b83948cab335336171d2f9ba29f160de05